ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/viewmodel/Event.kt

```diff
@@ -0,0 +1,30 @@
+package org.wordpress.android.imageeditor.viewmodel
+
+/**
+ * Used as a wrapper for data that is exposed via a LiveData that represents an event.
+ */
+open class Event<out T>(private val content: T) {
```

Comment:
I think it'd be nice to start thinking about extracting these common classes into a module so we don't need to create copies. It's out of scope for now, but we should keep it in mind so we don't forget about it.